src/components/Editor.vue

```diff
@@ -612,6 +616,7 @@ export default {
 			this.emit('update:content', {
 				markdown: proseMirrorMarkdown,
 			})
+			this.isEmptyContent = editor.state.doc.nodeSize <= 4
```

Comment:
What is the background for that fixed number of 4? I'd assume we can also check the text content length directly?

Reply:
text content length is checked by the count of `nodeSize` from `editor.state.doc`, checked with Max

Reply:
> text content length is checked by the count of nodeSize from editor.state.doc

Yes, that is what the code does. I would still be curious about the reasoning, I assume there is one ...